cl_dll/saytext.cpp

```diff
@@ -221,6 +221,20 @@ void CHudSayText :: SayTextPrint( const char *pszBuf, int iBufSize, int clientIn
 		ConsolePrint( pszBuf );
 		return;
 	}
+	char timestamp[16];
+	std::time_t curtime = std::time(nullptr);
+	auto written = std::strftime(timestamp, sizeof(timestamp), "[%H:%M:%S] ", std::localtime(&curtime));
+
+	if(written > 0)
```

Comment:
You can just call ConsolePrint twice instead of concatenating the string
```cpp
ConsolePrint( timestamp );
ConsolePrint( pszBuf );
```

Reply:
oh yea, thanks